### PR TITLE
RM-262845 RM-262844 Release over_react 5.2.0

### PR DESCRIPTION
**Pull request by rmconsole-readonly-wk**
Pull Requests included in release:
* Minor changes:
	* [FED-1945 Support prop conversion in OverReact to simplify RMUI converted props](https://github.com/Workiva/over_react/pull/920)
* Patch changes:
	* [Remove publish action that always fails due to dry run warnings](https://github.com/Workiva/over_react/pull/925)
	* [FED-2854 Fix num keys incorrectly linting as not unique](https://github.com/Workiva/over_react/pull/928)
	* [FED-2855 Fix Analyzer Plugin error when typing keys](https://github.com/Workiva/over_react/pull/930)
	* [FED-2960 Update the null safety migration documentation](https://github.com/Workiva/over_react/pull/932)

Requested by: @sydneyjodon-wk

Diff Between Last Tag and Proposed Release: https://github.com/Workiva/over_react/compare/5.1.2...Workiva:release_over_react_5.2.0
Diff Between Last Tag and New Tag: https://github.com/Workiva/over_react/compare/5.1.2...5.2.0

The logs for the request that created this PR can be found [here](https://w-rmconsole.appspot.com/release/release_event/6449952306233344/logs/)
This pull request can be recreated by clicking [here](https://w-rmconsole.appspot.com/api/v2/rosie/reTriggerReleaseEvents/6449952306233344/?repo_name=Workiva%2Fover_react&pull_number=933)

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,12 @@
 # OverReact Changelog
+
+## 5.2.0
+- [#920] Support prop conversion in OverReact to simplify RMUI converted props
```
Comment (greglittlefield-wf) [inline CHANGELOG.md:4]: #nit I think this changelog could be a little more specific and reference the added APIs
```suggestion
- [#920] Support conversion of prop values via annotations: `@ConvertProp`, `@convertJsMapProp`, `@convertJsRefProp`
```

**Review by greglittlefield-wf — DISMISSED** [review]
+10

**Review by aaronlademann-wf — APPROVED** [review]

**Review by greglittlefield-wf — APPROVED** [review]
+10

**Comment by sydneyjodon-wk** [comment]
@Workiva/release-management-p

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**